src/app/meshes/views/MeshListView.vue

```diff
@@ -61,7 +61,7 @@
                       >
                         <template #icon>
                           <KIcon
-                            color="var(--black-400)"
+                            :color="KUI_COLOR_BACKGROUND_NEUTRAL_STRONGER"
```

Comment:
issue: icons (SVGs) should use `color-text` tokens instead of `color-background`

Reply:
Fixed in https://github.com/kumahq/kuma-gui/pull/1314/commits/c790d358059f3fcab222a0d22a51c8b81d9790de.